x-pack/plugins/profiling/server/routes/stacktrace.ts

```diff
@@ -115,9 +114,9 @@ export function runLengthDecodeReverse(input: Buffer, outputSize?: number): numb
   const output: number[] = new Array(size);
 
   let idx = 0;
-  for (let i = input.length - 1; i >= 1; i -= 2) {
```

Comment:
this starts at l-1 and stops at 1, but the other starts at 0 and runs up until length - 1. is there a functional difference?

Reply:
The functional difference is reversed vs non-reversed (see the change in the function name).

Imagine the input is [3,1], both sides will produce [1,1,1].
With [3,1,1,4], left produces [4,1,1,1] and right produces [1,1,1,4].

Reply:
ok, I see that they are now equivalent but in reverse. not sure if anything changed there or I've misread it before, but looks good now :)